File: src/main/java/FuNWallet.Client/FuNWallet.Client/Models/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Linq;

namespace FuNWallet.Client.Models
{
    public class Student
    {
        public string FullName { get; set; }
        public string ID { get; set; }
        public string Nationality { get; set; }
        public string School { get; set; } = "FUN University";
        public double Balance { get; set; }
        public double Debt { get; set; }

        public static Student FromServerResponse(string response)
        {
            string[] data = response.Split("_");
            var studentID = data[0];
            var name = data[1];
            var nationality = data[2];

            return new Student
            {
                ID = studentID,
                FullName = name,
                Nationality = nationality
            };
        }
    }
}

File: AppControl/Student.java
package AppControl;

public class Student {
    private String studentID;
    private String name;
    private String nationality;

    public Student(String input) {
        String[] data = input.split("_");
        studentID = data[0];
        name = data[1];
        nationality = data[2];
    }

    public String getStudentID() {
        return studentID;
    }

    public String getName() {
        return name;
    }

    public String getNationality() {
        return nationality;
    }

    public void setStudentID(String studentID) {
        this.studentID = studentID;
    }
}
